addons/repair/stringtable.xml

```diff
@@ -677,6 +705,7 @@
         </Key>
         <Key ID="STR_ACE_Repair_engineerSetting_RepairSpecialistOnly">
             <English>Repair Specialist only</English>
+            <German>Nur Mechatroniker</German>
```

Comment:
... wie ist denn die Übersetzung in Vanilla-Arma für "Repair Specialist"?
Ich glaube nicht, dass das "Mechatroniker" ist.